api/client/gro_client.py

```diff
@@ -302,8 +302,24 @@ def get_geo_centre(self, region_id):
         """
         return lib.get_geo_centre(self.access_token, self.api_host, region_id)
 
+
+    def get_geojsons(self, region_id, descendant_region_level_id=None):
```

Comment:
Could we make the `descendant_region_level` kwarg name consistent with `get_descendant_regions`?

`descendant_level` is what it uses

Reply:
done

Reply:
This would be a breaking change, changing the name in `get_descendant_regions`

I meant change `get_geojsons` since it's an addition and nothing depends on it yet

Reply:
right it's a kwarg not an arg! done